### PR TITLE
Package oraft.0.1.1

### PR DESCRIPTION
**Pull request by komamitsu**
### `oraft.0.1.1`
Raft consensus algorithm implemented in OCaml
Raft consensus algorithm implemented in OCaml

---
* Homepage: https://github.com/komamitsu/oraft
* Source repo: git+https://github.com/komamitsu/oraft.git
* Bug tracker: https://github.com/komamitsu/oraft/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by mseri** [comment]
It is failing with
```
#=== ERROR while compiling oraft.0.1.1 ========================================#
# context              2.0.7 | linux/x86_64 | ocaml-base-compiler.4.10.1 | pinned(https://github.com/komamitsu/oraft/archive/0.1.1.tar.gz)
# path                 ~/.opam/4.10/.opam-switch/build/oraft.0.1.1
# command              ~/.opam/4.10/bin/dune runtest -p oraft -j 72
# exit-code            1
# env-file             ~/.opam/log/oraft-29-e22673.env
# output-file          ~/.opam/log/oraft-29-e22673.out
### output ###
# File "unit_test/dune", line 3, characters 18-27:
# 3 |  (libraries oUnit fileutils oraft))
#                       ^^^^^^^^^
# Error: Library "fileutils" not found.
# Hint: try:
#   dune external-lib-deps --missing -p oraft -j 72 @runtest

- File "unit_test/dune", line 3, characters 18-27:
- 3 |  (libraries oUnit fileutils oraft))
-                       ^^^^^^^^^
- Error: Library "fileutils" not found.
- Hint: try:
-   dune external-lib-deps --missing -p oraft -j 72 @runtest
```

File: packages/oraft/oraft.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "Raft consensus algorithm implemented in OCaml"
+description: "Raft consensus algorithm implemented in OCaml"
+maintainer: "komamitsu@gmail.com"
+authors: "Mitsunori Komatsu"
+license: "Apache"
+homepage: "https://github.com/komamitsu/oraft"
+bug-reports: "https://github.com/komamitsu/oraft/issues"
+depends: [
+  "dune"
```
Comment (mseri) [inline packages/oraft/oraft.0.1.1/opam:10]:
```suggestion
  "ocaml" {>= "4.05.0"}
  "dune"
```
`Hashtbl.find_opt` was introduced in 4.05.

Comment (mseri) [reply packages/oraft/oraft.0.1.1/opam:11]: @komamitsu can you upstream this change?

**Comment by komamitsu** [comment]
@mseri Thanks! I added the change to the project https://github.com/komamitsu/oraft/pull/23/files

**Comment by komamitsu** [comment]
```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - oraft.0.1.1:
            https://github.com/komamitsu/oraft/archive/0.1.1.tar.gz (Bad
        checksum,
            expected md5=752327af2d02014113fa149fe029c16f)
```
Oops. I'll update this PR again later

**Comment by kit-ty-kate** [comment]
Could you return the fix upstream? Your `dune-project` file tells us this package requires at least dune 2.0.

Thanks!

**Pull request merged**